lib/core/include/irods/getRodsEnv.h

```diff
@@ -55,13 +55,19 @@ typedef struct RodsEnvironment {
 
     // =-=-=-=-=-=-=-
     // override of plugin installation directory
-    char irodsPluginHome[MAX_NAME_LEN];
+    __attribute__((deprecated("Use irodsPluginDirectory"))) char irodsPluginHome[MAX_NAME_LEN];
+    // TODO Hmm, adding a new member will result in an ABI break. Users will have to
+    // recompile their C/C++ clients. If we don't add the new member, then users can't
+    // migrate their clients to the new variable.
```

Comment:
Dumb question: I thought an ABI break only occurred when members are removed or added in the middle of the struct? Hmm... I guess if the `RodsEnvironment` is a member of another struct it would cause a misalignment...

Reply:
Any changes to a data structure can cause ABI issues because there's a change in the layout, size, etc.

I want to make the iRODS 5 member available, but I'm not confident it's possible without breaking the ABI (forcing people to recompile code) or using come special compiler flag (yuck).